amundsen_application/static/js/ducks/dashboard/reducer.ts

```diff
@@ -21,14 +21,20 @@ export function getDashboard(payload: { uri: string, searchIndex?: string, sourc
 export function getDashboardFailure() {
   return {
     type: GetDashboard.FAILURE,
-    payload: {}
+    payload: {
+      isLoading: false,
```

Comment:
I copied the pattern used in by the table metadata API, which is probably outdated. Is the Data Preview our best example of how to handle failure? I'll try using that pattern instead.

Reply:
Yeah we don't have consistency, but recalling we had a previous issue with the fact that logic for failures was handled in so many different places and settled on `sagas` as the ideal  place in case the failure response needs special logic. But if doesn't need special logic and the failure case is a default value of some kind I think it's also fair to hardcode that in the reducer, like setting something back to an initial or default state.

Yes I'd say the preview logic is our best example for the most complex failure case we currently have, which requires processing the failure response.